src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -31,25 +31,25 @@ export function UnderReview(response, name) {
             </p>
           ) : null}
 
-          {response ? (
-            <>
-              <ul className="claim-list">
-                <li>
-                  <strong>Date received</strong>
-                  <br />
-                  <span>{format(new Date(), 'MMM d, yyyy')}</span>
-                </li>
-              </ul>
+          {/* {response ? ( */}
+          {/*  <> */}
+          <ul className="claim-list">
+            <li>
+              <strong>Date received</strong>
+              <br />
+              <span>{format(new Date(), 'MMM d, yyyy')}</span>
```

Comment:
A definition list would be a little more semantic here.

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/my-education-benefits/components/confirmation/ConfirmationApproved.jsx#L39

Reply:
Updated HTML tags for the Date received from UL to DL.